Source/WTF/wtf/RunLoop.h

```diff
@@ -201,10 +200,6 @@ class WTF_CAPABILITY("is current") RunLoop final : public SerialFunctionDispatch
         Function<void()> m_function;
     };
 
-private:
-    class Holder;
-    static ThreadSpecific<Holder>& runLoopHolder();
-
     class DispatchTimer final : public TimerBase {
```

Comment:
Change it to `class DispatchTimer final : public TimerBase, public ThreadSafeRefCounted<DispatchTimer>`.